app/components/affiliate-link-page/logo-cloud.js

```diff
@@ -9,15 +9,15 @@ import stripeImage from '/assets/images/company-logos/stripe.png';
 import appleImage from '/assets/images/company-logos/apple.png';
 import porscheImage from '/assets/images/company-logos/porsche-1.png';
 import bookingImage from '/assets/images/company-logos/bookingcom.png';
-import supabaseImage from '/assets/images/company-logos/supabase.png';
+import supabaseImage from '/assets/images/company-logos/supabase-dark.svg';
 import deliverooImage from '/assets/images/company-logos/deliveroo.png';
-import dockerImage from '/assets/images/company-logos/docker-company-logo.svg';
+import dockerImage from '/assets/images/company-logos/docker-dark.svg';
 import metaImage from '/assets/images/company-logos/meta.png';
 import grabImage from '/assets/images/company-logos/grab.png';
 import salesforceImage from '/assets/images/company-logos/salesforce.png';
 import substackImage from '/assets/images/company-logos/substack.png';
-import tencentImage from '/assets/images/company-logos/tencent-white.svg';
-import adobeImage from '/assets/images/company-logos/adobe.png';
+import tencentImage from '/assets/images/company-logos/tencent-dark.svg';
+import adobeImage from '/assets/images/company-logos/adobe-company-logo.svg';
```

Comment:
This whole thing seems inconsistent - some logos have `-dark`, some have `-company-logo`, some have `company-logo-dark`. I actually don't think we even need this many versions - could we not add `filter: brightness(0);` (that makes the image black except for transparent areas), and then combine that with `opacity: 0.4`?